WalletWasabi.Fluent/Views/AddWallet/AddWalletPageView.axaml

```diff
@@ -14,33 +16,45 @@
                  EnableBack="{Binding EnableBack}"
                  IsBusy="{Binding IsBusy}"
                  EnableNext="False">
-    <UniformGrid Rows="1" HorizontalAlignment="Center">
-      <UniformGrid.Styles>
-        <Style Selector="c|TileButton">
-          <Setter Property="Margin" Value="10" />
-          <Setter Property="IconSize" Value="50" />
-        </Style>
-      </UniformGrid.Styles>
-      <c:TileButton Text="Create a new wallet"
-                    Classes="action doubleline"
-                    Icon="{StaticResource add_regular}"
-                    Command="{Binding CreateWalletCommand}"
-                    IsDefault="{Binding IsActive}" />
+    <Panel>
+      <i:Interaction.Behaviors>
+        <ir:AdaptiveBehavior>
+          <ir:AdaptiveClassSetter MinWidth="0" MaxWidth="600" MaxWidthOperator="LessThanOrEqual" ClassName="small" />
```

Comment:
To fix this:
![image](https://user-images.githubusercontent.com/16364053/204505600-a51709f1-b2ab-401d-873c-7aef758476be.png)


```suggestion
          <ir:AdaptiveClassSetter MinWidth="0" MaxWidth="650" MaxWidthOperator="LessThanOrEqual" ClassName="small" />
```